training_program/tests/tests_serializers.py

```diff
@@ -0,0 +1,37 @@
+'''Unit tests for training_program serializers.'''
+from django.test import TestCase
+
+import training_program.serializers as serializers
+import training_program.models as models
+
+
+class TestProgramCategorySerializer(TestCase):
+    '''Unit tests for serializer of programcategory.'''
+    def test_fields_equal(self):
+        '''Serializer should return fields of ProgramCategory correctly.'''
+        programcategory = models.ProgramCategory()
```

Comment:
```suggestion
        category = models.ProgramCategory()
```
Given the context, It's ok to name the variable as `category`.